### DISCUSSION TITLE
dirtyFields.has() function is removed since 6.3.1. Is there any altenative?

### DISCUSSION THREAD

**Discussion by anhquande** [react-hook-form/react-hook-form#2569] [Q&A] [answered]
I want to check if a field is dirty. 
Before version 6.3.1, i can write as follows
  

    const {formState} = useContext(formContext);

    const dirtyFields = formState.dirtyFields;

    const dirty = dirtyFields.has(name);

However, after update to 6.3.1, the has() function is removed from dirtyFields object.

Is there any other way to get the same result ?

**Comment by bluebill1049** [comment, marked as answer]
you can use `lodash.has`, we have convert into `object` for consistency.
